### DISCUSSION TITLE
Where’s Reunion 0.8?

### DISCUSSION THREAD

**Discussion by wjk** [microsoft/WindowsAppSDK#860] [Q&A] [answered]
I have read on several sites, including Microsoft’s official Windows developer blog, that Reunion 0.8 was released today. However, the latest version on GitHub and on NuGet as of this writing is still 0.5.7. What’s going on?

**Comment by orcmid** [comment]
I saw that number in a report by Mary Jo Foley, but I missed the MS Build Keynote and saw no other significant mention of Project Reunion.  I also see from March, [when 0.5 was announced](https://www.zdnet.com/article/microsofts-project-reunion-hits-the-0-5-milestone/) that 0.8 was targeted for Q2 2021, with 1.0 for Q4 2021.  That is the same roadmap on the README here.  So, fingers crossed, aye?

**Reply by riverar** [reply]
Am told we'll hear more tomorrow. https://twitter.com/andrewclinick/status/1397274523389489153

Related: #859

**Comment by andrewleader** [comment]
We are working on getting the 0.8 Preview released as soon as possible. Stay tuned to the Discussions, that's where we'll announce when it's live!

**Reply by orcmid** [reply]
Yes, the Andrew Clinick and Thomas Fennel session (which is very humorous) did cover it.  0.8 preview late May (a.k.a real-soon -now) with release in (squinting closely) June maybe.   I think "release" was the word used.

**Reply by riverar** [reply]
> Yes, the Andrew Clinick and Thomas Fennel session (which is very humorous) did cover it.  0.8 preview late May (a.k.a real-soon -now) with release in (squinting closely) June maybe.   I think "release" was the word used.

The release word was also used at Build to refer to 0.8 yet here we are. I trust they'll release it, these are good people, but I wish there was more transparency and honesty here.

**Reply by orcmid** [reply]
> I trust they'll release it, these are good people, but I wish there was more transparency and honesty here.

It strikes me that the GitHub microsoft/ProjectReunion is a great example of code in the open, but not of an open-source managed project.  The single Project kanban is evidence of that, there being negligible reflection of internal/external coordination and mileposts.  I can understand the necessity of that separation.  It is gratifying that the published roadmap seems to be achievable, within reason.  Especially since we're not paying the bills ;).

I don't have anything useful to say about early-adopter syndrome and the drama that can arise around a version 1.0 death-march that may have functionality gaps and no prediction of a time-line for remedies.   Historically, it is a good idea to stay off the skinny branches and keep the safety net in place.  Don't create irreversible dependencies based on speculative remedies.  Just don't.  Treat like beta. Seriously.

**Reply by yoshiask** [reply]
This [tweet from @WindowsDev](https://twitter.com/windowsdev/status/1397953313560272902) went out 30 minutes ago, but there's still no sign of 0.8 anywhere. I understand there are probably deployment issues or something, but I'd expect that at least the PR team would be notified that the release is not available yet. I wish you luck with whatever's happening

**Reply by riverar** [reply]
@yoshiask Looks like it was deleted, so seems like they're on top of it now.

**Reply by orcmid** [reply]
This too: https://blogs.windows.com/windowsdeveloper/2021/05/25/the-windows-developers-guide-to-microsoft-build-2021/

Some lessons are  being learned (again)?

**Reply by riverar** [reply]
`0.8-preview` is [on NuGet](https://www.nuget.org/packages/Microsoft.ProjectReunion.WinUI/0.8.0-preview) now so this discussion is moot now.

**Reply by yoshiask** [reply]
> `0.8-preview` is [on NuGet](https://www.nuget.org/packages/Microsoft.ProjectReunion.WinUI/0.8.0-preview) now so this discussion is moot now.

Awesome! But the [`Microsoft.ProjectReunion` package](https://www.nuget.org/packages/Microsoft.ProjectReunion/) is still on 0.5.7?

**Reply by andrewleader** [reply]
Hey folks, we're still not live yet, as you've noticed, we have all the packages on 0.8-preview *except* the main package. We're working on getting the last issue resolved (we had a NuGet packaging error - it installs fine but somehow an additional duplicate file got included that we didn't notice and NuGet.org does some extra validation that Visual Studio doesn't). We'll get it fixed and published as soon as possible, and then we'll post the announcements/docs/etc!

**Comment by andrewleader** [comment, marked as answer]
The 0.8 Preview is now available! https://docs.microsoft.com/windows/apps/project-reunion/preview-channel
